app/javascript/components/TeacherSetDetails.jsx

```diff
@@ -340,15 +340,15 @@ export default function TeacherSetDetails(props) {
             id="ts-order-set"
             textAlign="center"
             noSpace
-            level="two"
+            level="h2"
             size="secondary"
             text="Order Set"
           />
           <Heading
             id="ts-available-copies"
             textAlign="center"
             size="callout"
-            level="four"
+            level="h4"
             text={AvailableCopies()}
           />
```

Comment:
This information can be set using the `subtitle` prop of the previous `Heading` component.